### DISCUSSION TITLE
Inability to detect specific free ac

### DISCUSSION THREAD

**Discussion by Naaf83** [RaresAil/homebridge-gree-ac#10] [Q&A]
Hi,
the plugin works great for me but I'm unable to detect a specific gree ac with.

7/10/2022, 9:30:41 AMGree Ac Platform Bridge starting
7/10/2022, 9:30:44 AMGree Ac Platform BridgeLoaded plugin 'homebridge-gree-ac-platform'
7/10/2022, 9:30:44 AMGree Ac Platform BridgeLoading 1 platforms...
7/10/2022, 9:30:44 AMGree Ac Platform BridgeBridge is running on port 51836.
7/10/2022, 9:30:44 AMGree Ac Platform BridgeGree ACsAdding new accessory: c6704db8
7/10/2022, 9:30:44 AMGree Ac Platform BridgeGree ACsAdding new accessory: c63c1786
7/10/2022, 9:30:44 AMGree Ac Platform BridgeGree ACsAdding new accessory: c63c0dbf
**7/10/2022, 9:30:44 AMGree Ac Platform BridgeGree ACsAdding new accessory:** 
7/10/2022, 9:30:44 AMGree Ac Platform BridgeGree ACsAdding new accessory: c6854722

The ac appears on the log without its MAC identifier (Line 8)

Any way I can make it appear?

**Comment by RaresAil** [comment]
And the AC is not visible in the app?

**Comment by Naaf83** [comment]
It appears on gree app

**Reply by RaresAil** [reply]
I no longer have a Gree AC and I can't add support for it, but if you know programming, you can modify the plugin and open a PR for it
